client/src/game/ui/reactive/view/currentPlayerAreaView.ts

```diff
@@ -51,30 +51,28 @@ export function onChanged(
     return;
   }
   const currentPlayerHand = globals.state.ongoingGame.hands[currentPlayerIndex];
-  const isLocked = handRules.isLocked(
-    currentPlayerHand,
-    globals.state.ongoingGame.deck,
-  );
+  const isLocked =
```

Comment:
its a misleading variable name because you can be locked without playing the hyphenated conventions